plugins/ssrmatching/ssrmatching.ml

```diff
@@ -56,7 +56,10 @@ let _ =
       optwrite = debug }
 let pp s = !pp_ref s
 let { Goptions.get = option_LegacyFoUnif } =
-  Goptions.declare_bool_option_and_ref ~key:["SsrMatching";"LegacyFoUnif"] ~value:false ()
+  Goptions.declare_bool_option_and_ref ~key:["SsrMatching";"LegacyFoUnif"]
+    ~depr:{Deprecation.since=Some "9.2";
+            note=Some "Use this flag only to port scripts from 9.1 to 9.2, do not leave it in final scripts." }
```

Comment:
Maybe the flag can be mentionned in the changelog entry?

Reply:
done